### PR TITLE
Update Navigator UI

### PR DESCRIPTION
**Pull request by zobione**
Adding Overlap Analysis, Segment Dashboard View, and new Chart Styles

### CODE REVIEW

**Comment by thomastosoni** [comment]
Hello @zobione 
https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages

File: app/react/src/containers/Audience/Segments/Dashboard/Charts/Overlap.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+    const nextLocation = {
+      pathname,
+      search: updateSearch(currentSearch, params, SEGMENT_QUERY_SETTINGS)
+    };
+
+    history.push(nextLocation);
+  }
+
+  renderDatePicker() {
```
Comment (ndussart) [inline app/react/src/containers/Audience/Segments/Dashboard/Charts/Overlap.js]: Overlap component is not linked to dates, please remove everything related that handle dates range

File: app/react/src/containers/Audience/Segments/Dashboard/Charts/Overlap.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        params: {
+          organisationId,
+          segmentId
+        }
+      }
+    } = this.props;
+    const datamartId = defaultDatamart(organisationId).id;
+
+    Modal.confirm({
+      title: 'Create an Overlap Analysis',
```
Comment (ndussart) [inline app/react/src/containers/Audience/Segments/Dashboard/Charts/Overlap.js]: Not internationalized ?

File: app/react/src/containers/Audience/Segments/List/AudienceSegmentsTable.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ class AudienceSegmentsTable extends Component {
       },
       loadAudienceSegmentsDataSource
     } = this.props;
-
+    console.log('mounting');
```
Comment (ndussart) [inline app/react/src/containers/Audience/Segments/List/AudienceSegmentsTable.js]: no console.log shall remain

File: app/react/src/containers/Creative/DisplayAds/List/DisplayAdsList.js [hunk 4]
```diff
@@ -42,15 +42,17 @@ class CreativeDisplayTable extends Component {
       },
       fetchCreativeDisplay
     } = this.props;
-
+    console.log('components mounting');
```
Comment (ndussart) [inline app/react/src/containers/Creative/DisplayAds/List/DisplayAdsList.js]: no console.log will be allowed on board

File: app/react/src/containers/Logo/Logo.js [hunk 5]
```diff
@@ -49,14 +49,13 @@ class Logo extends Component {
...
     const logoUrl = logoBlob ? URL.createObjectURL(logoBlob) : null;  // eslint-disable-line no-undef
-
     return (
       <div className="mcs-logo-placeholder">
         { mode === 'inline' &&
           <div className="mcs-logo" >
             <Link to="/" id="logo">
               {!!logoBlob && <img alt="logo" src={logoUrl} />}
-              {!logoBlob && <img alt="logo" src={mediarithmicsLogo} />}
+              {!logoBlob && <img alt="logo" src={logoUrl ? logoUrl : mediarithmicsLogo} />}
```
Comment (ndussart) [inline app/react/src/containers/Logo/Logo.js]: Instead maybe you can just write:
```html
<Link to="/" id="logo">
  <img alt="logo" src={logoUrl ? logoUrl : mediarithmicsLogo} />
</Link>
```

File: app/react/src/containers/Menu/NavigatorMenu.js [hunk 6]
```diff
@@ -58,8 +60,10 @@ class NavigatorMenu extends Component {
   }
 
   onClick = ({ key }) => {
+    // console.log('has clicked on item');
```
Comment (ndussart) [inline app/react/src/containers/Menu/NavigatorMenu.js]: delete unused code

File: app/react/src/state/Audience/Partitions/sagas.js [hunk 7]
```diff
@@ -33,7 +34,29 @@ function* loadAudiencePartitionsList({ payload }) {
...
     if (filter.types) { options.types = filter.types; }
 
-    const response = yield call(AudiencePartitionsService.getPartitions, organisationId, datamartId, options);
+    const initialOptions = {
+      ...getPaginatedApiParam(1, 1)
+    };
+
+    let allCalls;
+
+    if (isInitialRender) {
```
Comment (ndussart) [inline app/react/src/state/Audience/Partitions/sagas.js]: I don't think it is the saga's responsibility to know that it is an initial render
I would create 2 separates actions and dispatch the initial one on componentDidMount

File: app/react/src/state/Audience/Segments/sagas.js [hunk 8]
```diff
@@ -84,19 +144,116 @@ function* loadAudienceSegmentList({ payload }) {
...
+        const overlapData = yield call(DataFileService.getDatafileData, micsUri);
+        formatedResponse = {
+          ...overlapData,
+          hasOverlap: true
+        };
+      } else if (response.data[0].status === 'PENDING' || response.data[0].status === 'RUNNING') {
+
+        let responseStatus = response.data[0].status;
+
+        while (responseStatus !== 'SUCCEEDED') {
```
Comment (ndussart) [inline app/react/src/state/Audience/Segments/sagas.js]: Implement a way to exit loop

**Pull request merged**